### PR TITLE
Fix + Update (libse/HtmlUtil.cs, src/Forms/OCRSpellCheck.cs)

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/ManagedBitmap.cs [hunk 1]
```diff
@@ -17,9 +17,8 @@ public ManagedBitmap(string fileName)
...
             {
                 byte[] buffer = new byte[1024];
-                MemoryStream fd = new MemoryStream();
-                Stream fs = File.OpenRead(fileName);
-                using (Stream csStream = new GZipStream(fs, CompressionMode.Decompress))
+                using (var fd = new MemoryStream())
+                using (Stream csStream = new GZipStream(File.OpenRead(fileName), CompressionMode.Decompress))
                 {
                     int nRead;
                     while ((nRead = csStream.Read(buffer, 0, buffer.Length)) > 0)
```
Comment (xylographe) [inline libse/ManagedBitmap.cs]: Perhaps, replace this loop with `csStream.CopyTo(fd)` ?

```
    using (var fd = new MemoryStream())
    using (var csStream = new GZipStream(File.OpenRead(fileName), CompressionMode.Decompress))
    {
        csStream.CopyTo(fd)
        buffer = fd.ToArray();
    }
```

Comment (niksedk) [inline libse/ManagedBitmap.cs]: Yes, CopyTo would be nice - is a new .net 4 feature, right?

Comment (ivandrofly) [inline libse/ManagedBitmap.cs]: :+1:

Comment (xylographe) [inline libse/ManagedBitmap.cs]: Yes, `CopyTo` method was added in .NET Framework 4.
`CopyToAsync` was added in 4.5.

**Comment by ivandrofly** [comment]
:+1:  @niksedk

File: libse/StripableText.cs [hunk 2]
```diff
@@ -215,7 +215,7 @@ public void FixCasing(List<string> namesEtc, bool changeNameCases, bool makeUppe
             if (makeUppercaseAfterBreak && StrippedText.Contains(new[] { '.', '!', '?', ':', ';', ')', ']', '}', '(', '[', '{' }))
```
Comment (niksedk) [inline libse/StripableText.cs]: could you also make this a private static var (as you did earlier for another new array)?

**Comment by ivandrofly** [comment]
Done

**Comment by ivandrofly** [comment]
Var with same name :)

**Comment by ivandrofly** [comment]
Ready /cc @niksedk

**Comment by niksedk** [comment]
ok, cool :)
perhaps we can improve the naming later (often the hardest part)

**Comment by ivandrofly** [comment]
:wink:

**Pull request merged**